app/src/main/java/com/duckduckgo/mobile/android/duckduckgo/ui/bookmarks/BookmarkModel.java

```diff
@@ -0,0 +1,99 @@
+package com.duckduckgo.mobile.android.duckduckgo.ui.bookmarks;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.duckduckgo.mobile.android.duckduckgo.domain.bookmark.Bookmark;
+
+import java.util.UUID;
+
+/**
+ * Created by fgei on 6/12/17.
+ */
+
+public class BookmarkModel implements Bookmark, Parcelable {
```

Comment:
we could also just remove the interface and rename the BookmarkModel -> Bookmark and keep BookmarkEntity